src/renderer/gdi/paint.cpp

```diff
@@ -22,7 +22,7 @@ using namespace Microsoft::Console::Render;
 HRESULT GdiEngine::StartPaint() noexcept
 {
     // If we have no handle, we don't need to paint. Return quickly.
-    RETURN_HR_IF(S_FALSE, INVALID_HANDLE_VALUE == _hwndTargetWindow);
+    RETURN_HR_IF(S_FALSE, !_IsWindowValid());

```

Comment:
If a USER/GDI call to open or create an object like a window fails, that call should return a null handle (HWND for windows). Yet `this->_hwndTargetWindow` is initialized to INVALID_HANDLE_VALUE. Out of caution, I changed this test to use `this->_IsWindowValid()` which tests against both null and INVALID_HANDLE_VALUE.